OutOfSchool/OutOfSchool.WebApi/Services/IProviderService.cs

```diff
@@ -18,6 +18,13 @@ public interface IProviderService
         /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
         Task<ProviderDto> Create(ProviderDto providerDto);
 
+        /// <summary>
+        /// Add entity to the database.
+        /// </summary>
+        /// <param name="dto">Entity to add.</param>
+        /// <returns>The <see cref="Task"/> that represents the asynchronous operation, containing the <see cref="ProviderDto"/>.</returns>
+        Task<ProviderDto> CreateV2(ProviderDto dto);
```

Comment:
Please consider if there are any benefits to introduce separate `...V2` interface, and either use it as base in `IProvideService` (not a good idea, just in case) or implement it in specific provider classes?

Reply:
Okey, sounds really better, I'll rewrite